idl/chromadb/proto/coordinator.proto

```diff
@@ -208,6 +208,7 @@ message FlushCollectionCompactionRequest {
   int64 log_position = 3;
   int32 collection_version = 4;
   repeated FlushSegmentCompactionInfo segment_compaction_info = 5;
+  uint64 total_records_post_compaction = 6;
```

Comment:
nit: can remove `post_compaction` suffix since it is implicit

Reply:
Agreed that it's implicit but wanted to be extremely explicit in the `SysDB` to ensure no confusion and wanted this field to align in naming.